lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -87,6 +87,39 @@ class AwsCompileEventBridgeEvents {
           required: ['inputTemplate'],
           additionalProperties: false,
         },
+        retryPolicy: {
+          type: 'object',
+          properties: {
+            maximumEventAge: {
+              type: 'integer',
+              minimum: 60,
+              maximum: 86400,
+            },
+            maximumRetryAttempts: {
+              type: 'integer',
+              minimum: 0,
+              maximum: 185,
+            },
+          },
+        },
+        deadLetterConfig: {
+          type: 'object',
+          properties: {
+            targetArn: {
+              anyOf: [
+                {
+                  type: 'string',
+                  pattern:
+                    '^arn:aws[a-zA-Z-]*:elasticfilesystem:[a-z]{2}((-gov)|(-iso(b?)))?-[a-z]+-[1-9]{1}:[0-9]{12}:access-point/fsap-[a-f0-9]{17}$',
```

Comment:
I'm afraid this is not the correct pattern for `deadLetterConfig`'s arn - this one is only for EFS's access point. Maybe we can just use `{ $ref: '#/definitions/awsArn' }` here? Do you see any incompatibility for that property?

Reply:
Yeah, I didn't see that "elasticfilesystem" there, sorry!